### PR TITLE
Class 03

### PR DESCRIPTION
**Pull request by carlox7**
got projects to append to page

### CODE REVIEW

**Comment by carlox7** [comment]
this assignment took about 3 hours. I spent time catching up on portfolio 2 assignment and then started functionality of my click to hide event handler.

File: app.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ function Project (proj){
 }
 
 Project.prototype.toHtml = function() {
```
Comment (JudyVue) [inline app.js:12]: **TODO FOR LOST POINTS**
This is a top level comment that you need to separate your files into separate folders, i.e. js files in ``scripts``, css files in ``styles``.

File: index.html [hunk 2]
```diff
@@ -25,13 +25,13 @@
...
     </div>
-    <div class="about">
+    <div id="profile" class="about">
       <h1>About Me:</h1>
       <p>
         Ang Lorem Ipsum ay ginagamit na modelo ng industriya ng pagpriprint at pagtytypeset. Ang Lorem Ipsum ang naging regular na modelo simula pa noong 1500s, noong may isang di kilalang manlilimbag and kumuha ng galley ng type at ginulo ang pagkaka-ayos nito upang makagawa ng libro ng mga type specimen. Nalagpasan nito hindi lang limang siglo, kundi nalagpasan din nito ang paglaganap ng electronic typesetting at nanatiling parehas. Sumikat ito noong 1960s kasabay ng pag labas ng Letraset sheets na mayroong mga talata ng Lorem Ipsum, at kamakailan lang sa mga desktop publishing software tulad ng Aldus Pagemaker ginamit ang mga bersyon ng Lorem Ipsum.
       </p>
     </div>
-    <div class="projects">
+    <div class="projectsTemp">
```
Comment (JudyVue) [inline index.html:34]: Make HTML attribute names not camel-case, i.e. Line 34 would be:

```
<div class="projects-temp">
```

Doing camel-case doesn't break things, but it's professional convention to not camel-case in HTML. Confusing, I know, but just a professional tip.

File: index.html [hunk 3]
```diff
@@ -25,13 +25,13 @@
...
       <h1>Projects</h1>
       <section id="work" class="currentProjects">
```
Comment (JudyVue) [inline index.html:36]: Same no-camel-case tip applies to Line 36.

**Pull request merged**